src/plone/volto/tests/data/5.json

```diff
@@ -0,0 +1,35 @@
+[
+  {
+    "type": "p",
+    "children": [
+      {
+        "text": "Paragraph "
+      },
+      {
+        "type": "strong",
+        "children": [
+          {
+            "text": ""
```

Comment:
Those text nodes are intentional, see https://github.com/plone/plone.volto/blob/9207feb01a71ad62c2b52cd9161dacbfada313df/src/plone/volto/slate/html2slate.py#L385-L412

Reply:
@davisagli Slate needs those empty text nodes to know where to place a cursor, when you click at the beginning or end of an "inline block element".

Reply:
@tiberiuichim I'm going to take a closer look at this, because it's not what blocks-conversion-tool is doing and I need to understand whether it is producing output that will be a problem.

My gut feeling is that even if these empty text nodes are needed at runtime, they should be added implicitly when the editor loads and filtered out when the data is serialized, rather than being part of the stored data. They have no semantic value.

Reply:
@davisagli The intention was to perform a perfect roundtrip. Indeed, a normalization at JS volto-slate level is probably better. 

It is possible that slatejs itself is less sensitive to those missing text nodes in the values, I used to get a lot of crashes when the value that was injected as a prop (I don't remember my exact test scenarios, it's possible that I was trying to push it too much).

Reply:
@davisagli I test, the padding is no longer needed. I'll disable it.

Reply:
@tiberiuichim nice!